master/docs/examples/hello.cfg

```diff
@@ -1,72 +1,71 @@
 # -*- python -*-
 # ex: set syntax=python:
 
-from buildbot.plugins import *
+from buildbot.plugins import buildslave
+from buildbot.plugins import changes
+from buildbot.plugins import util
+from buildbot.plugins import steps
```

Comment:
"Google's Style Guidelines" are not exactly followed here.

What is followed is [PEP-8](https://www.python.org/dev/peps/pep-0008/) with Twisted twisted names for methods and functions.